android/KMAPro/kMAPro/src/main/java/com/tavultesoft/kmapro/MainActivity.java

```diff
@@ -176,7 +176,10 @@ protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
     context = this;
 
-    SentryAndroid.init(context);
+    SentryAndroid.init(context, options -> {
+      options.setRelease("release-"+com.tavultesoft.kmea.BuildConfig.VERSION_NAME);
+      options.setEnvironment(com.tavultesoft.kmea.BuildConfig.VERSION_ENVIRONMENT);
```

Comment:
If 3rd party app does a debug build and uses release build of KMEA, won't this set the environment to "release"? (when the app environment is "local")

Reply:
But it's a release build of KMEA -- even if they are testing it. So I think that's appropriate.